config/defaults.go

```diff
@@ -46,6 +47,7 @@ func GetDefaults() Defaults {
 		YPServer:  "https://directory.owncast.online",
 
 		WebServerPort:  8080,
+		WebServerIP:    "127.0.0.1",
```

Comment:
Ditto.  I should probably de-duplicate these two sets default values, sorry about that.